packages/twilio-worker-auth/src/twilioWorkerAuthMiddleware.ts

```diff
@@ -76,35 +102,19 @@ export const getAuthorizationMiddleware = (authTokenLookup: (accountSid: string)
       if (!isWorker(tokenResult) || isGuest(tokenResult)) {
         return unauthorized(res);
       }
-
       req.user = twilioUser(tokenResult.worker_sid, tokenResult.roles);
       return next();
     } catch (err) {
       console.error('Token authentication failed: ', err);
     }
   }
 
-  if (authorization.startsWith('Basic')) {
-    if (canAccessResourceWithStaticKey(req.originalUrl, req.method)) {
-      try {
-        const staticSecretKey = `STATIC_KEY_${accountSid}`;
-        const staticSecret = process.env[staticSecretKey];
-        const requestSecret = authorization.replace('Basic ', '');
-
-        const isStaticSecretValid =
-          staticSecret &&
-          requestSecret &&
-          crypto.timingSafeEqual(Buffer.from(requestSecret), Buffer.from(staticSecret));
-
-        if (isStaticSecretValid) {
-          req.user = twilioUser(`account-${accountSid}`, []);
-          return next();
-        }
-      } catch (err) {
-        console.error('Static key authentication failed: ', err);
-      }
-    }
-  }
+  if (canAccessResourceWithStaticKey(req.originalUrl, req.method) && authenticateWithStaticKey(req)) return next();
```

Comment:
I think you need to add the `/resources/import` path to `canAccessResourceWithStaticKey`?